src/main/java/com/cricket/project/controller/PlayerController.java

```diff
@@ -0,0 +1,61 @@
+package com.cricket.project.controller;
+
+import com.cricket.project.exception.PlayerException;
+import com.cricket.project.model.Player;
+import com.cricket.project.model.PlayerScoreCard;
+import com.cricket.project.service.impl.PlayerServiceImpl;
+import com.cricket.project.service.impl.ScoreCardServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/players")
+public class PlayerController {
+
+    @Autowired
+    private PlayerServiceImpl playerService;
+
+    @Autowired
+    private ScoreCardServiceImpl scoreCardService;
+
+    @PostMapping
+    public ResponseEntity<String> addPlayer(@RequestBody Player player) {
+        String result = playerService.addPlayer(player);
+        return new ResponseEntity<>(result, HttpStatus.CREATED);
+    }
+
+    @GetMapping("/all")
+    public ResponseEntity<List<Player>> getPlayerList() throws PlayerException {
+        List<Player> players = playerService.getPlayerList();
+        return new ResponseEntity<>(players, HttpStatus.OK);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Player> getPlayerById(@PathVariable("id") int id) {
+        try {
+            Player player = playerService.getPlayerById(id);
+            return new ResponseEntity<>(player, HttpStatus.OK);
```

Comment:
you can create a utility method foe this which takes an object as an input and create ok response entity from it please refer RB MetadataApi